templates/README.template

```diff
@@ -1,112 +1,216 @@
 [![npm][npm]][npm-url]
+[![node][node]][node-url]
 [![deps][deps]][deps-url]
-[![test][test]][test-url]
+[![tests][tests]][tests-url]
 [![coverage][cover]][cover-url]
 [![chat][chat]][chat-url]
 
 <div align="center">
   <!-- replace with accurate logo e.g from https://worldvectorlogo.com/ -->
-  <img width="200" height="200" src="https://cdn.worldvectorlogo.com/logos/javascript.svg">
+  <img width="140" height="140" src="https://cdn.worldvectorlogo.com/logos/javascript.svg">
```

Comment:
Now it will render like this:

`[img] [hspace][img][hspace]`

This would be better:

`[hspace][img][hspace] [hspace][img][hspace]`